manifests/r/Rufus/Rufus/3.19/Rufus.Rufus.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Rufus.Rufus
+PackageVersion: "3.19"
+MinimumOSVersion: 10.0.0.0
+InstallerType: appx
+PackageFamilyName: Rufus_maxm3x3sw7c4t
```

```suggestion
InstallerType: portable
```
This is a candidate for portable apps which will be accepted once 1.3 goes GA and there are no issues with it.